test/serverTest.js

```diff
@@ -379,9 +379,9 @@ for (const supportedVersion of mc.supportedVersions) {
           entityId: client.id,
           levelType: 'default',
           gameMode: 1,
-          previousGameMode: 255,
+          previousGameMode: version.version >= 755 ? 0 : 255,
           worldNames: ['minecraft:overworld'],
-          dimensionCodec: (version.version >= 735 ? mcData.loginPacket.dimension : { name: '', type: 'compound', value: { dimension: { type: 'list', value: { type: 'compound', value: [w] } } } }),
+          dimensionCodec: version.version >= 755 ? mcData.loginPacket.dimensionCodec : (version.version >= 735 ? mcData.loginPacket.dimension : { name: '', type: 'compound', value: { dimension: { type: 'list', value: { type: 'compound', value: [w] } } } }),          dimension: (version.version >= 735 ? mcData.loginPacket.dimension : 0),
```

Comment:
you got dimension twice there, look at the end of the line

Reply:
OHHHHHH